sys-devel/elftoolchain/elftoolchain-0.7.1-r1.ebuild

```diff
@@ -40,16 +47,17 @@ _bmake() {
 
 src_compile() {
 	export MAKESYSPATH="${BROOT}"/usr/share/mk/bmake
-	_bmake -C common
-	_bmake -C libelf
+	_bmake
 }
 
 src_install() {
-	doheader common/elfdefinitions.h
-	doheader libelf/{gelf,libelf}.h
-
-	dolib.so libelf/libelf.so.1
-	dosym libelf.so.1 /usr/$(get_libdir)/libelf.so
-
-	dodoc README
+	_bmake \
+		DESTDIR="${D}" \
+		BINDIR="${EPREFIX}"/usr/bin/${CHOST}-elftoolchain \
```

Comment:
Clang/LLVM do /usr/lib/llvm/11/bin/clang, but obviously they don't bother with tuples in the paths. I'm honestly not that bothered about what we do here as long as it doesn't collide with anything else or confuse people.

-elftoolchain is pretty unique although I feel like we're abusing that part of the tuple. @SoapGentoo?